dripline/extensions/modbus/ethernet_modbus_service.py

```diff
@@ -0,0 +1,160 @@
+try:
+    import pymodbus
+    from pymodbus.client import ModbusTcpClient
+    from pymodbus.payload import BinaryPayloadDecoder
+except ImportError:
+    pass
+
+import scarab
+
+from dripline.core import calibrate, Entity, Service, ThrowReply
+
+import logging
+logger = logging.getLogger(__name__)
+
+__all__ = []
+
+
+__all__.append('EthernetModbusService')
+class EthernetModbusService(Service):
+    '''
+    Service for connectivity to ModbusTCP instruments built on pymodbus library.
+    '''
+    def __init__(self,
+                 ip_address,
+                 indexing='protocol',
+                 wordorder='big',
+                 byteorder='big',

```

Comment:
The byte order is fixed by the modbus standard see https://github.com/pymodbus-dev/pymodbus/pull/2456